msf_updates/src/position_msf/init_yaw.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+
+PACKAGE = 'msf_updates'
+import roslib;roslib.load_manifest(PACKAGE)
+import rospy
+from geometry_msgs.msg import Vector3
+
+import dynamic_reconfigure.client
+
+client = dynamic_reconfigure.client.Client("msf_gps_pose_estimator/position_sensor", timeout=30)
+
+# Write init_yaw MSF parameter from orientation message
+def callback_orientation(data):
```

Comment:
Add subscribers for Quaternion (specifying a yaw rotation), and for Float64 (check that mag_imu publishes that).